### PR TITLE
Modify words that may cause ambiguity in device-plugins.md

### PR DESCRIPTION
**Pull request by pokerfaceSad**
The `/var/lib/kubelet/pod-resources/kubelet.sock` is required by device monitoring agent but not device plugin.
This word  `plugin` is ambiguous.

plugin -> device monitoring agent

### CODE REVIEW

**Comment by pokerfaceSad** [comment]
More description in #25938

**Comment by tengqm** [comment]
/approve
Needs technical lgtm.

**Comment by pokerfaceSad** [comment]
@tengqm How to get /lgtm label? Need assign this PR to somebody?

**Comment by tengqm** [comment]
@pokerfaceSad My secret is patience.

**Comment by pokerfaceSad** [comment]
@tengqm Alright, thx for your secret😅
I am a little hurry because I have submit this problem a month ago in another PR #25009 but have no response at all. 
And I think it is an obvious ambiguity.

**Comment by bradtopol** [comment]
Hi @pokerfaceSad Thanks for you contribution. I will try and get you an expedited technical review.
/assign
/cc @mikebrow

**Comment by bradtopol** [comment]
hi @mikebrow Can you do a technical review of this?  Simply add an lgtm if it is technically accurate. Thanks!

**Review by mikebrow — APPROVED** [review]
LGTM

**Comment by sftim** [comment]
/lgtm
echoing https://github.com/kubernetes/website/pull/25937#pullrequestreview-562100025

**Comment by pokerfaceSad** [comment]
> Hi @pokerfaceSad Thanks for you contribution. I will try and get you an expedited technical review.
> /assign
> /cc @mikebrow

Thanks.

**Pull request merged**
